README.md

```diff
@@ -0,0 +1,132 @@
+# Attestation Service
+
+Attestation Service (AS for short) is a general function set that can strategically verify TEE evidence.
+In the confidential containers architecture, the functions provided by the AS need to run in a security environment out of the guest node.
+When remote attestation is required (for example, the [Attestation Agent](https://github.com/confidential-containers/attestation-agent) in the guest node requests the key from [KBS](https://github.com/confidential-containers/kbs)),
+the AS receives the evidence generated by the Attestation Agent (AA for short) and verify whether the HW-TEE signature of the evidence is correct
+and whether the TCB status (including hardware informations and software measurements) meets expectations.
+The verification of TCB status is completed by AS's [Policy Engine](#policy-engine), and can be customized for policy configuration.
+
+AS can be integrated into other confidential resources provider services as a library crate, such as the Key Broker Service,
+or run independently as a service program.
+When the AS runs as an independent program, you need to call the API of the AS through grpc, https, or other protocols.
+
+# Components
+
+## Library Crate
+
+Import AS in `Cargo.toml` of your project:
+
+```toml
+attestation-service = { git = "https://github.com/confidential-containers/attestation-service" }
```

Comment:
Do we need to add `branch = "master"` here? If new branch is pushed in the future

Reply:
Done.